lib/activerecord-bitemporal/bitemporal.rb

```diff
@@ -113,14 +113,25 @@ def ast(klass: nil, table: nil)
           option = ::ActiveRecord::Bitemporal.merge_by(self[klass] || {})
 
           table = klass.arel_table unless table
-          table = option[:through].arel_table if option[:through].present?
+          if option[:through].present?
+            klass = option[:through]
+            table = klass.arel_table
+          end
```

Comment:
Are there any reasons for this change ?
I don't think it's a good idea to substitute a dummy argument again.

Reply:
`klass = option[:through]` is required to use the predicate builder for the klass for typecasting correctly.

https://github.com/kufu/activerecord-bitemporal/blob/4050dbb3b9bd963de94057b3b45eda1f11dc95c2/lib/activerecord-bitemporal/bitemporal.rb#L124-L125
https://github.com/kufu/activerecord-bitemporal/blob/4050dbb3b9bd963de94057b3b45eda1f11dc95c2/lib/activerecord-bitemporal/bitemporal.rb#L133-L135

Related #57.

Reply:
OK, Thanks.